apps/gp2-frontend/src/hooks/onboarding.tsx

```diff
@@ -0,0 +1,83 @@
+import { gp2 as gp2Validation } from '@asap-hub/validation';
+import { gp2 as gp2Routing } from '@asap-hub/routing';
+import { useLocation } from 'react-router-dom';
+import { useUserById } from '../users/state';
+
+const { isUserOnboardable } = gp2Validation;
+
+const orderedSteps = [
+  'Core Details',
+  'Background',
+  'GP2 Groups',
+  'Additional Details',
+  'Preview',
+] as const;
+
+export const stepToHref: Record<typeof orderedSteps[number], string> = {
+  'Core Details': gp2Routing.onboarding({}).coreDetails({}).$,
+  Background: gp2Routing.onboarding({}).background({}).$,
```

Comment:
Both Background and Preview should be in a string, no? Or there's a particular reason for it?

Reply:
it's just formatting